diff --unified --recursive --text a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi	2022-01-09 22:55:34.000000000 +0000
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi	2022-07-26 12:05:28.817220415 +0100
@@ -233,7 +233,7 @@
 				<0 0 0 2 &pcie0_intc 1>,
 				<0 0 0 3 &pcie0_intc 2>,
 				<0 0 0 4 &pcie0_intc 3>;
-		max-link-speed = <1>;
+		max-link-speed = <2>;
 		msi-map = <0x0 &its 0x0 0x1000>;
 		phys = <&pcie_phy 0>, <&pcie_phy 1>,
 		       <&pcie_phy 2>, <&pcie_phy 3>;
diff --unified --recursive --text a/scripts/dtc/include-prefixes/arm64/rockchip/rk3399.dtsi b/scripts/dtc/include-prefixes/arm64/rockchip/rk3399.dtsi
--- a/scripts/dtc/include-prefixes/arm64/rockchip/rk3399.dtsi	2022-01-09 22:55:34.000000000 +0000
+++ b/scripts/dtc/include-prefixes/arm64/rockchip/rk3399.dtsi	2022-07-26 12:05:28.817220415 +0100
@@ -233,7 +233,7 @@
 				<0 0 0 2 &pcie0_intc 1>,
 				<0 0 0 3 &pcie0_intc 2>,
 				<0 0 0 4 &pcie0_intc 3>;
-		max-link-speed = <1>;
+		max-link-speed = <2>;
 		msi-map = <0x0 &its 0x0 0x1000>;
 		phys = <&pcie_phy 0>, <&pcie_phy 1>,
 		       <&pcie_phy 2>, <&pcie_phy 3>;
